ansible/roles/k3s/defaults/main.yml

```diff
@@ -3,3 +3,4 @@ k3s_version: "v1.31.0+k3s1"
 k3s_selinux_release: v1.6.latest.1
 k3s_selinux_rpm_version: 1.6-1
 k3s_helm_version: v3.11.0
+k3s_bootstrap_token_expiry: 20m
```

Comment:
Does this even need to be so long: At the point we generate the token we're immediately going to use it, so I'd have thought 10 mins is fine - might depend on how hard recovering a broken cluster is though?